dependencies/dependencies.go

```diff
@@ -0,0 +1,23 @@
+package dependencies
+
+type HostLookup interface {
+	Hosts() []string
+	Watch() <-chan struct{}
+}
+
+type BucketLookup interface {
+	Lookup(ctx context.Context, orgID platform.ID, name string) (platform.ID, bool)
+	LookupName(ctx context.Context, orgID platform.ID, id platform.ID) string
+}
+
+type OrganizationLookup interface {
+	Lookup(ctx context.Context, name string) (platform.ID, bool)
+	LookupName(ctx context.Context, id platform.ID) string
+}
+
+type Dependencies struct {
+	Reader             Reader
+	BucketLookup       BucketLookup
+	OrganizationLookup OrganizationLookup
+	Metrics            *metrics
+}
```

Comment:
Does this need to be part of this PR?If so can you add comments to describe why/how it changing?

Reply:
no, just a dirty repo mistake.  this isn't even a proper design, just something I laid out for discussion